commands/review_apps/disable.js

```diff
@@ -0,0 +1,77 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+const api = require('../../lib/api')
+const KolkrabbiAPI = require('../../lib/kolkrabbi-api')
+
+module.exports = {
+  topic: 'reviewapps',
+  command: 'disable',
+  description: 'disable review apps or settings on an existing pipeline',
+  help: `Example:
+
+    $ heroku reviewapps:disable -p mypipeline -a myapp --autodeploy
+    Disabling auto deployment ...
+    Configuring pipeline... done`,
+  needsApp: false,
+  needsAuth: true,
+  wantsOrg: false,
+  args: [],
+  flags: [
+    {
+      name: 'pipeline',
```

Comment:
There is no need to define the pipeline flag like that. `cli-engine-heroku` includes this which allows us no needing to specify the name or description.

You'll need to import:

```js
const { flags } = require('cli-engine-heroku')
```

And then use it:

```js
flags: [
    flags.pipeline({ name: 'pipeline', required: false, hasValue: true })
  ]
```

Here's an example https://github.com/heroku/heroku-pipelines/blob/master/commands/pipelines/transfer.js#L72.

This applies to all commands that use the pipeline flag.

Reply:
I've updated in both `disable` and `enable`.